requirements/scenarios.md

```diff
@@ -1,21 +1,21 @@
-# Notary v2 Signing Scenarios
+# Notary Project Signing Scenarios
 
 As containers and cloud native artifacts become the common unit of deployment, users want to know the artifacts in their environments are authentic and unmodified.
 
-These Notary v2 scenarios define end-to-end scenarios for signing artifacts in a generalized way, storing and moving them between OCI compliant registries, validating them with various artifact hosts and tooling.
-Notary v2 focuses on the signing of content, enabling e2e workflows, without specifying what those workflows must be.
+These Notary Project scenarios define end-to-end scenarios for signing artifacts in a generalized way, storing and moving them between OCI compliant registries, validating them with various artifact hosts and tooling.
+Notary Project focuses on the signing of content, enabling e2e workflows, without specifying what those workflows must be.
 
 By developing a generalized solution, artifact authors may develop their unique artifact types, allowing them to leverage Notary for signing and OCI Compliant registries for distribution.
 
 ## Scenarios
 
-Notary v2 aims to solve the core issue of trusting content within, and across registries.
-There are many elements of an end to end scenario that are not implemented by Notary v2, rather enabled because the content is verifiable.
+Notary Project aims to solve the core issue of trusting content within, and across registries.
+There are many elements of an end to end scenario that are not implemented by Notary Project specification implementation, rather enabled because the content is verifiable.
 
 ### Scenario #0: Build, Publish, Consume, Enforce Policy, Deploy
 
-To put Notary v2 in context, the following end-to-end scenario is outlined.
-The blue elements are the scope of Notary v2, with the other elements providing generic references to other projects or products that demonstrate how Notary v2 may be utilized.
+To put Notary Project in context, the following end-to-end scenario is outlined.
+The blue elements are the scope of Notary Project, with the other elements providing generic references to other projects or products that demonstrate how Notary Project may be utilized.
```

Comment:
The blue elements are the scope of the Notary Project, with the other elements providing generic references to other projects or products that demonstrate how the Notary Project may be utilized.